src/components/DataExplorerApp/VisualizationContainer.tsx

```diff
@@ -33,6 +34,8 @@ const BarChartSettingsSchema = z.object({
   }),
 });
 
+const LineChartSettingsSchema = BarChartSettingsSchema;
```

Comment:
Create a separate LineChartSettingsSchema. With Schemas, it's usually better to make the building blocks reusable (e.g. xAxis or yAxis) but make the high-level schemas unique. Even if they have the same properties at this moment, very soon they will start to diverge. LineCharts and BarCharts will have very different settings.